packages/vite/src/node/server/ws.ts

```diff
@@ -157,6 +157,9 @@ export function createWebSocketServer(
       socket.send(JSON.stringify(bufferedError))
       bufferedError = null
     }
+    socket.on('error', (e) => {
+      config.logger.info(`WebSocket error:\n${e.stack || e.message}`)
```

Comment:
```suggestion
      config.logger.error(`WebSocket error:\n${e.stack || e.message}`)
```

Should we make this an error? Since it's an actual error that's happening.

Reply:
I think we should also move the `socket.on('error'` call before the first `socket.send` call. Also we could follow this pattern when calling `logger.error` too:

https://github.com/vitejs/vite/blob/a71c305b207a749949d4b379cf4dfad43e56f233/packages/vite/src/node/server/middlewares/proxy.ts#L71-L74

(Passing the timestamp and error)